### DISCUSSION TITLE
My GitHub account suddenly stopped and I was unable to access it

### DISCUSSION THREAD

**Discussion by ALI-WOSABY** [community/community#86631] [Other Feature Feedback, Questions, & Ideas]
I have an account on GitHub called Aliwosabi666 for my work with companies. The problem is that I did the verification using the authenticator app program. My problem is that I accidentally deleted the link through which the verification is done and the verification code was sent to me. Now I cannot access my account. I tried more than once and it was of no use, knowing that The email is correct: programmeurweb.a@gmail.com
  As well as the password, I want you to delete the verification method for me by sending the code to my email so that I can log into my account. Currently, I am in a situation and I have not been able to upload my work with companies on it. Please remove the verification through the authenticator app program and return the old method, which is sending the code to my email.

**Comment by ALI-WOSABY** [comment]
How can I contact the GitHub team so that I can recover my account and continue my work, please?

**Reply by karasowles** [reply]
I'd recommend submitting a ticket to GitHub Support. They'll know what they can, and can't, help with: https://support.github.com/contact/account

**Reply by ALI-WOSABY** [reply]
I sent the ticket to them two days ago, and to this day my problem has not been resolved and my work has been suspended because of that

**Comment by ALI-WOSABY** [comment]
Please, how many days does it take for a response to the ticket from JibHub? I applied approximately three days ago and have not received any response from them until today, and my work has stopped

**Comment by samus-aran** [comment]
Hi @ALI-WOSABY ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
